src/FormModelInterface.php

```
@@ -25,21 +25,36 @@ interface FormModelInterface extends ModelInterface
+     *         'Username must contain only word characters.',
+     *     ],
+     *     'email' => [
+     *         'Email address is invalid.',
+     *     ]
+     * ]
+     * ```
+     *
+     * @phpstan-return array<string, array<int, string>|string>
      */
-    public function clearError(string $property = null): void;
+    public function getErrors(bool $first = false): array;
```

PHPMD (2.15.0)

[error] The method getErrors has a boolean flag argument $first, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)